pyitlib/discrete_random_variable.py

```diff
@@ -4744,7 +4744,7 @@ def _sanitise_array_input(X, fill_value=-1):
         else:
             X = X.filled()
     else:
-        X = np.array(X, copy=False)
+        X = np.array(X, copy=True)
```

Comment:
Why are we copying?

Reply:
> Why are we copying?

based on numpy 2.0 migration, need it as `ValueError: Unable to avoid copy while creating an array as requested `

Reply:
https://github.com/charris/numpy/blob/89ed6060ebdbe0207baab055c022fdbc155a6f15/doc/source/numpy_2_0_migration_guide.rst#adapting-to-changes-in-the-copy-keyword